tools/shared/src/main/scala/org/scalajs/core/tools/linker/LinkedClass.scala

```diff
@@ -61,6 +61,12 @@ final class LinkedClass(
   def encodedName: String = name.name
   def isExported: Boolean = classExports.nonEmpty
 
+  /** Names this class / module is exported under */
+  def exportNames: List[String] = classExports.collect {
```

Comment:
Consider calling this `topLevelExportNames`. This makes it clear that instance methods exports are not included. It also suggests that we can reuse that when we support exporting top-level functions.